.gitmodules

```diff
@@ -19,3 +19,6 @@
 	path = lib/base58-solidity
 	url = https://github.com/storyicon/base58-solidity
 	branch = v1.0.2
+[submodule "lib/@aave/aave-vault"]
+	path = lib/@aave/aave-vault
+	url = https://github.com/aave/Aave-Vault
```

Comment:
What should we do about failing CI?
I suggest we put forked aave repo in zkbob Github org, until it's merged into AAVE.